### ISSUE TITLE
2025-05-06, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#161]
# Today's Arxiv Papers

(1) [Scenethesis: A Language and Vision Agentic Framework for 3D Scene Generation](http://arxiv.org/abs/2505.02836v1), 2025-05-05

(2) [R1-Reward: Training Multimodal Reward Model Through Stable Reinforcement Learning](http://arxiv.org/abs/2505.02835v1), 2025-05-05

(3) [AutoLibra: Agent Metric Induction from Open-Ended Feedback](http://arxiv.org/abs/2505.02820v1), 2025-05-05

(4) [ReplaceMe: Network Simplification via Layer Pruning and Linear Transformations](http://arxiv.org/abs/2505.02819v1), 2025-05-05

(5) [Knowing You Don't Know: Learning When to Continue Search in Multi-round RAG through Self-Practicing](http://arxiv.org/abs/2505.02811v1), 2025-05-05

(6) [Towards Quantifying the Hessian Structure of Neural Networks](http://arxiv.org/abs/2505.02809v1), 2025-05-05

(7) [Generating HomeAssistant Automations Using an LLM-based Chatbot](http://arxiv.org/abs/2505.02802v1), 2025-05-05

(8) [HSplitLoRA: A Heterogeneous Split Parameter-Efficient Fine-Tuning Framework for Large Language Models](http://arxiv.org/abs/2505.02795v1), 2025-05-05

(9) [Giving Simulated Cells a Voice: Evolving Prompt-to-Intervention Models for Cellular Control](http://arxiv.org/abs/2505.02766v1), 2025-05-05

(10) [Bye-bye, Bluebook? Automating Legal Procedure with Large Language Models](http://arxiv.org/abs/2505.02763v1), 2025-05-05

(11) [Knowledge Graphs for Enhancing Large Language Models in Entity Disambiguation](http://dx.doi.org/10.1007/978-3-031-77844-5_9), 2025-05-05

(12) [FormalMATH: Benchmarking Formal Mathematical Reasoning of Large Language Models](http://arxiv.org/abs/2505.02735v1), 2025-05-05

(13) [Enhancing LLMs' Clinical Reasoning with Real-World Data from a Nationwide Sepsis Registry](http://arxiv.org/abs/2505.02722v1), 2025-05-05

(14) [Less is More: Efficient Weight Farcasting with 1-Layer Neural Network](http://arxiv.org/abs/2505.02714v1), 2025-05-05

(15) [Voila: Voice-Language Foundation Models for Real-Time Autonomous Interaction and Voice Role-Play](http://arxiv.org/abs/2505.02707v1), 2025-05-05

(16) [Structure Causal Models and LLMs Integration in Medical Visual Question Answering](http://dx.doi.org/10.1109/TMI.2025.3564320), 2025-05-05

(17) [Exploring LLM-Powered Role and Action-Switching Pedagogical Agents for History Education in Virtual Reality](http://arxiv.org/abs/2505.02699v1), 2025-05-05

(18) [AI Standardized Patient Improves Human Conversations in Advanced Cancer Care](http://arxiv.org/abs/2505.02694v1), 2025-05-05

(19) [Predicting Movie Hits Before They Happen with LLMs](http://arxiv.org/abs/2505.02693v1), 2025-05-05

(20) [Sailing AI by the Stars: A Survey of Learning from Rewards in Post-Training and Test-Time Scaling of Large Language Models](http://arxiv.org/abs/2505.02686v1), 2025-05-05

(21) [A Survey on Progress in LLM Alignment from the Perspective of Reward Design](http://arxiv.org/abs/2505.02666v1), 2025-05-05

(22) [A Survey of Slow Thinking-based Reasoning LLMs using Reinforced Learning and Inference-time Scaling Law](http://arxiv.org/abs/2505.02665v1), 2025-05-05

(23) [A Note on Statistically Accurate Tabular Data Generation Using Large Language Models](http://arxiv.org/abs/2505.02659v1), 2025-05-05

(24) [Enhancing Chemical Reaction and Retrosynthesis Prediction with Large Language Model and Dual-task Learning](http://arxiv.org/abs/2505.02639v1), 2025-05-05

(25) [Parameter-Efficient Fine-Tuning with Attributed Patch Semantic Graph for Automated Patch Correctness Assessment](http://arxiv.org/abs/2505.02629v1), 2025-05-05

(26) [Detect, Classify, Act: Categorizing Industrial Anomalies with Multi-Modal Large Language Models](http://arxiv.org/abs/2505.02626v1), 2025-05-05

(27) [LLaMA-Omni2: LLM-based Real-time Spoken Chatbot with Autoregressive Streaming Speech Synthesis](http://arxiv.org/abs/2505.02625v1), 2025-05-05

(28) [Towards Cross-Modality Modeling for Time Series Analytics: A Survey in the LLM Era](http://arxiv.org/abs/2505.02583v1), 2025-05-05

(29) [EMORL: Ensemble Multi-Objective Reinforcement Learning for Efficient and Flexible LLM Fine-Tuning](http://arxiv.org/abs/2505.02579v1), 2025-05-05

(30) [Recursive Decomposition with Dependencies for Generic Divide-and-Conquer Reasoning](http://arxiv.org/abs/2505.02576v1), 2025-05-05

(31) [Evaluating Contrastive Feedback for Effective User Simulations](http://dx.doi.org/10.1145/3726302.3730189), 2025-05-05

(32) [The Turing Test Is More Relevant Than Ever](http://arxiv.org/abs/2505.02558v1), 2025-05-05

(33) [Bielik v3 Small: Technical Report](http://arxiv.org/abs/2505.02550v1), 2025-05-05

(34) [Large Language Model Partitioning for Low-Latency Inference at the Edge](http://arxiv.org/abs/2505.02533v1), 2025-05-05

(35) [Unveiling the Landscape of LLM Deployment in the Wild: An Empirical Study](http://arxiv.org/abs/2505.02502v1), 2025-05-05

(36) [Automating Automotive Software Development: A Synergy of Generative AI and Formal Methods](http://arxiv.org/abs/2505.02500v1), 2025-05-05

(37) [Beyond the model: Key differentiators in large language models and multi-agent services](http://dx.doi.org/10.30574/wjarr.2025.26.1.1295), 2025-05-05

(38) [SEFE: Superficial and Essential Forgetting Eliminator for Multimodal Continual Instruction Tuning](http://arxiv.org/abs/2505.02486v1), 2025-05-05

(39) [El Agente: An Autonomous Agent for Quantum Chemistry](http://arxiv.org/abs/2505.02484v1), 2025-05-05

(40) [Automated Hybrid Reward Scheduling via Large Language Models for Robotic Skill Learning](http://arxiv.org/abs/2505.02483v1), 2025-05-05

(41) [Tevatron 2.0: Unified Document Retrieval Toolkit across Scale, Language, and Modality](http://arxiv.org/abs/2505.02466v1), 2025-05-05

(42) [Incentivizing Inclusive Contributions in Model Sharing Markets](http://arxiv.org/abs/2505.02462v1), 2025-05-05

(43) [Colombian Waitresses y Jueces canadienses: Gender and Country Biases in Occupation Recommendations from LLMs](http://arxiv.org/abs/2505.02456v1), 2025-05-05

(44) [Can LLM-Simulated Practice and Feedback Upskill Human Counselors? A Randomized Study with 90+ Novice Counselors](http://arxiv.org/abs/2505.02428v1), 2025-05-05

(45) [SymbioticRAG: Enhancing Document Intelligence Through Human-LLM Symbiotic Collaboration](http://arxiv.org/abs/2505.02418v1), 2025-05-05

(46) [Task-Oriented Semantic Communication in Large Multimodal Models-based Vehicle Networks](http://dx.doi.org/10.1109/TMC.2025.3564543), 2025-05-05

(47) [Estimating Commonsense Scene Composition on Belief Scene Graphs](http://arxiv.org/abs/2505.02405v1), 2025-05-05

(48) [Optimizing Chain-of-Thought Reasoners via Gradient Variance Minimization in Rejection Sampling and RL](http://arxiv.org/abs/2505.02391v1), 2025-05-05

(49) [RM-R1: Reward Modeling as Reasoning](http://arxiv.org/abs/2505.02387v1), 2025-05-05

(50) [EntroLLM: Entropy Encoded Weight Compression for Efficient Large Language Model Inference on Edge Devices](http://arxiv.org/abs/2505.02380v1), 2025-05-05

(51) [LAMeD: LLM-generated Annotations for Memory Leak Detection](http://arxiv.org/abs/2505.02376v1), 2025-05-05

(52) [Advancing Email Spam Detection: Leveraging Zero-Shot Learning and Large Language Models](http://arxiv.org/abs/2505.02362v1), 2025-05-05

(53) [An End-to-End Model For Logits Based Large Language Models Watermarking](http://arxiv.org/abs/2505.02344v1), 2025-05-05

(54) [VAEmo: Efficient Representation Learning for Visual-Audio Emotion with Knowledge Injection](http://arxiv.org/abs/2505.02331v1), 2025-05-05

(55) [From Course to Skill: Evaluating LLM Performance in Curricular Analytics](http://arxiv.org/abs/2505.02324v1), 2025-05-05

(56) [HyperTree Planning: Enhancing LLM Reasoning via Hierarchical Thinking](http://arxiv.org/abs/2505.02322v1), 2025-05-05

(57) [Invoke Interfaces Only When Needed: Adaptive Invocation for Large Language Models in Question Answering](http://arxiv.org/abs/2505.02311v1), 2025-05-05

(58) [Optimizing LLMs for Resource-Constrained Environments: A Survey of Model Compression Techniques](http://arxiv.org/abs/2505.02309v1), 2025-05-05

(59) [Generative Sign-description Prompts with Multi-positive Contrastive Learning for Sign Language Recognition](http://arxiv.org/abs/2505.02304v1), 2025-05-05

(60) [A survey of agent interoperability protocols: Model Context Protocol (MCP), Agent Communication Protocol (ACP), Agent-to-Agent Protocol (A2A), and Agent Network Protocol (ANP)](http://arxiv.org/abs/2505.02279v1), 2025-05-04

(61) [Real-time Spatial Retrieval Augmented Generation for Urban Environments](http://arxiv.org/abs/2505.02271v1), 2025-05-04

(62) [Personalisation or Prejudice? Addressing Geographic Bias in Hate Speech Detection using Debias Tuning in Large Language Models](http://arxiv.org/abs/2505.02252v1), 2025-05-04

(63) [SEval-Ex: A Statement-Level Framework for Explainable Summarization Evaluation](http://arxiv.org/abs/2505.02235v1), 2025-05-04

(64) [LLM-Guided Probabilistic Program Induction for POMDP Model Estimation](http://arxiv.org/abs/2505.02216v1), 2025-05-04

(65) [An Empirical Study of Qwen3 Quantization](http://arxiv.org/abs/2505.02214v1), 2025-05-04

(66) [Leveraging LLMs to Automate Energy-Aware Refactoring of Parallel Scientific Codes](http://arxiv.org/abs/2505.02184v1), 2025-05-04

(67) [Measuring Hong Kong Massive Multi-Task Language Understanding](http://arxiv.org/abs/2505.02177v1), 2025-05-04

(68) [Identifying Legal Holdings with LLMs: A Systematic Study of Performance, Scale, and Memorization](http://arxiv.org/abs/2505.02172v1), 2025-05-04

(69) [A New HOPE: Domain-agnostic Automatic Evaluation of Text Chunking](http://dx.doi.org/10.1145/3726302.3729882), 2025-05-04

(70) [Incorporating Legal Structure in Retrieval-Augmented Generation: A Case Study on Copyright Fair Use](http://arxiv.org/abs/2505.02164v1), 2025-05-04

(71) [Large Language Models are overconfident and amplify human bias](http://arxiv.org/abs/2505.02151v1), 2025-05-04

(72) [QiMeng-Xpiler: Transcompiling Tensor Programs for Deep Learning Systems with a Neural-Symbolic Approach](http://arxiv.org/abs/2505.02146v1), 2025-05-04

(73) [VECSR: Virtually Embodied Common Sense Reasoning System](http://arxiv.org/abs/2505.02144v1), 2025-05-04

(74) [Exploring the Potential of Offline RL for Reasoning in LLMs: A Preliminary Study](http://arxiv.org/abs/2505.02142v1), 2025-05-04

(75) [Efficient Multivariate Time Series Forecasting via Calibrated Language Models with Privileged Knowledge Distillation](http://arxiv.org/abs/2505.02138v1), 2025-05-04

(76) [Enhancing LLM Code Generation: A Systematic Evaluation of Multi-Agent Collaboration and Runtime Debugging for Improved Accuracy, Reliability, and Latency](http://arxiv.org/abs/2505.02133v1), 2025-05-04

(77) [Attention Mechanisms Perspective: Exploring LLM Processing of Graph-Structured Data](http://arxiv.org/abs/2505.02130v1), 2025-05-04

(78) [GRAIL: Graph Edit Distance and Node Alignment Using LLM-Generated Code](http://arxiv.org/abs/2505.02124v1), 2025-05-04

(79) [DriveAgent: Multi-Agent Structured Reasoning with LLM and Multimodal Sensor Fusion for Autonomous Driving](http://arxiv.org/abs/2505.02123v1), 2025-05-04

(80) [Adversarial Cooperative Rationalization: The Risk of Spurious Correlations in Even Clean Datasets](http://arxiv.org/abs/2505.02118v1), 2025-05-04

(81) [MemEngine: A Unified and Modular Library for Developing Advanced Memory of LLM-based Agents](http://arxiv.org/abs/2505.02099v1), 2025-05-04

(82) [LLM-OptiRA: LLM-Driven Optimization of Resource Allocation for Non-Convex Problems in Wireless Communications](http://arxiv.org/abs/2505.02091v1), 2025-05-04

(83) [Retrieval-augmented in-context learning for multimodal large language models in disease classification](http://arxiv.org/abs/2505.02087v1), 2025-05-04

(84) [LecEval: An Automated Metric for Multimodal Knowledge Acquisition in Multimedia Learning](http://arxiv.org/abs/2505.02078v1), 2025-05-04

(85) [Leveraging LLM Agents and Digital Twins for Fault Handling in Process Plants](http://arxiv.org/abs/2505.02076v1), 2025-05-04

(86) [What do Language Model Probabilities Represent? From Distribution Estimation to Response Prediction](http://arxiv.org/abs/2505.02072v1), 2025-05-04

(87) [RTV-Bench: Benchmarking MLLM Continuous Perception, Understanding and Reasoning through Real-Time Video](http://arxiv.org/abs/2505.02064v1), 2025-05-04

(88) [From Mind to Machine: The Rise of Manus AI as a Fully Autonomous Digital Agent](http://arxiv.org/abs/2505.02024v1), 2025-05-04

(89) [Wide & Deep Learning for Node Classification](http://arxiv.org/abs/2505.02020v1), 2025-05-04

(90) [R-Bench: Graduate-level Multi-disciplinary Benchmarks for LLM & MLLM Complex Reasoning Evaluation](http://arxiv.org/abs/2505.02018v1), 2025-05-04

(91) [MLLM-Enhanced Face Forgery Detection: A Vision-Language Fusion Solution](http://arxiv.org/abs/2505.02013v1), 2025-05-04

(92) [Testing Database Systems with Large Language Model Synthesized Fragments](http://arxiv.org/abs/2505.02012v1), 2025-05-04

(93) [Towards Safer Pretraining: Analyzing and Filtering Harmful Content in Webscale datasets for Responsible LLMs](http://arxiv.org/abs/2505.02009v1), 2025-05-04

(94) [Restoring Calibration for Aligned Large Language Models: A Calibration-Aware Fine-Tuning Approach](http://arxiv.org/abs/2505.01997v1), 2025-05-04

(95) [LLM-based Text Simplification and its Effect on User Comprehension and Cognitive Load](http://arxiv.org/abs/2505.01980v1), 2025-05-04

(96) [A Survey on Privacy Risks and Protection in Large Language Models](http://arxiv.org/abs/2505.01976v1), 2025-05-04

(97) [Analyzing Cognitive Differences Among Large Language Models through the Lens of Social Worldview](http://arxiv.org/abs/2505.01967v1), 2025-05-04

(98) [A Comprehensive Analysis for Visual Object Hallucination in Large Vision-Language Models](http://arxiv.org/abs/2505.01958v1), 2025-05-04

(99) [Generative AI in clinical practice: novel qualitative evidence of risk and responsible use of Google's NotebookLM](http://dx.doi.org/10.1038/s41433-025-03817-y), 2025-05-04

(100) [Embedding based retrieval for long tail search queries in ecommerce](http://dx.doi.org/10.1145/3640457.3688039), 2025-05-03
